packages/map-template/src/components/Search/Search.jsx

```diff
@@ -140,6 +140,15 @@ function Search({ onSetSize, isOpen }) {
         setSearchResults(locations);
         setFilteredLocations(locations);
         setShowNotFoundMessage(locations.length === 0);
+
+        // Handles updates to scroll buttons when the category changes.
+        // When a category changes, the scroll buttons need to have their enabled/disabled states updated.
+        // Since some categories might load before the DOM element is fully rendered, we listen for the 'transitionend' event.
+        // The 'transitionend' event is triggered when the DOM element changes its size, which can occur as a result of new categories being fetched.
+        // Upon completion of the size transition, the 'updateScrollButtons' function is triggered to handle the updated state.
+        searchRef.current?.addEventListener('transitionend', () => {
+            scrollButtonsRef?.current?.updateScrollButtons();
+        });
```

Comment:
Some cleanup and performance fixes are needed here.

- Don't apply the event listener if it is not needed (wrap in a check for `isKioskContext`).
- The `updateScrollMethods` method is triggered many, many times, and the event listener is not cleared up, causing memory leaks to happen. Consider applying a `{ once: true }` (but do check if that actually works) to the event listener or any other means to reduce memory leaks.

Reply:
Wrapped in `isKioskContext`
And added `once: true` - tested and seems to work. @ammapspeople